Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -71,6 +72,7 @@ open class CallViewModel: ObservableObject {
     @Published public var callingState: CallingState = .idle {
         didSet {
             handleRingingEvents()
+            audioRecorder.hasActiveCall = callingState != .idle
```

Comment:
this part can be tricky a bit for customers that won't use our view model, can we manage it internally instead in some way?

Reply:
This should be internal - whenever we enter a call we should update it in the SDK (shouldn't be available outside the LLC). Maybe we can manage it via the call controller?

Reply:
I updated it. WDYT?

Reply:
Actually, if we handle it internally we have an issue with the LobbyView, because the call isn't active yet.

Reply:
we can expose a public interface specifically for the lobby, while we still handle it internally. If you build your own lobby, you would have to manage it manually, which is not that bad.

Reply:
What i did was to have a parameter in the startListening that you can pass to ignore the activeCall existence.